internal/components/datasource/crd.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
What's the thinking behind introducing a committed version of the YAML file now?

Reply:
I need [to pass it to the tests](https://github.com/grafana/grafana/blob/34e8bf86a3d105e6107f65ecd0c71fa2a9b38ec8/internal/components/datasource/controller_test.go#L23): so that it will be installed when the `envtest.Environment` starts.